corehq/middleware.py

```diff
@@ -156,7 +157,14 @@ def process_response(self, request, response):
             response['Expires'] = "-1"
             response['Pragma'] = "no-cache"
         else:
+            content_type, _ = mimetypes.guess_type(request.path)
             response['Cache-Control'] = "max-age=31536000"
+            del response['Vary']
```

Comment:
So actually `del response[header]` doesn't raise an exception if that header isn't set, which you might not guess, but I just tried it out again to make sure.